admin/app/indexes/TagPages.scala

```diff
@@ -81,21 +81,25 @@ object TagPages {
     ("theguardian", "The Guardian"),
     ("theobserver", "The Observer"),
   )
-}
-
-class TagPages(implicit executionContext: ExecutionContext) extends GuLogging {
```

Comment:
https://github.com/guardian/frontend/pull/25176 removed the need for the `TagPages` to use `Future`s, so we don't need an `ExecutionContext` - and without any constructor parameters, the class has no interesting state, and all the methods in it can go to the singleton companion object above instead!